lab-dustin/test/readfile.test.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+const assert = require('assert');
+const read = require('../lib/readfile');
+
+describe('Read File Module', function(){
+  describe('#readFile()', function(){
+    it('should return first 8 bytes of yomamma.txt in hex format', function(done){
+      read.readData('yomamma.txt', function(err, data) {
+        //tests with data from callback
+        assert.ifError(err);
+        assert.strictEqual(data.hex, '596f206d');
+
+        done();
+      });
+    });
+    it('should return first 8 bytes of knockknock.txt in hex format', function(done){
+      read.readData('knockknock.txt', function(err, data) {
+        //tests with data from callback
+        assert.ifError(err);
+        assert.strictEqual(data.hex, '4b6e6f63');
+
+        done();
+      });
+    });
+    it('should return first 8 bytes of chucknorris.txt in hex format', function(done){
```

Comment:
These other two file tests are basically identical, so don't provide much value. Try other tests, e.g. "should return `null` if file doesn't exist".